source/deployment/advanced-permissions.rst

```diff
@@ -130,6 +141,30 @@ Example: In Team C, restrict public channel creation to Admins. As the default f
   ii. Select **Add Teams** to add Team B to the **Select teams to override permissions** list, locate Team B, and choose **Add**.   
   iii. In the **All Members** panel, in the **Manage Public Channels** section, uncheck the box for **Create Channels**.
   iv. In the **Team Administrators** panel, in the **Manage Public Channels** section, check the box for **Create Channels**.
+  
+Read only channels
+~~~~~~~~~~~~~~~~~~
+
+**Configure a channel so that members can post/reply/react but guests can only read and react.**
+
+1. Navigate to **System Console > User Management > Channels**.
+2. Select **Edit** next to the name of the channel you want to configure.
+3. In the **Create Posts** panel, uncheck **Guests**. 
+4. In the **Post Reactions** panel, uncheck **Guests** if required. 
+5. Choose **Save**. 
+
+The channel is available for all members and guests to access, but guests can only read messages and react to them.
+
+**Create an Announcement Channel where only channel admins are able to post.**
+
+1. Create a new channel (either Public or Private). 
```

Comment:
These are not rendering as a list because there are 2 step 2.'s

Reply:
Fixed, thank you!